### PR TITLE
Candidate for version 1

### PR DESCRIPTION
**Pull request by jabbett**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,108 @@
...
+* "Exit" and "Next" buttons on intermediate steps
+* "Done" button on the last step
+
+### Testing your tours
+
+Abraham loads tour definitions once when you start your server. If you'd like to iteratively test changes without restarting your server, use the Rails console to rerun the initializer:
+
+```
+???
+```
```
Comment (seouri) [inline README.md]: I couldn't find a way yet.

File: app/controllers/abraham_histories_controller.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+class AbrahamHistoriesController < ApplicationController
+  def create
+    @abraham_history = AbrahamHistory.new(abraham_history_params)
+    @abraham_history.creator_id = current_user
```
Comment (seouri) [inline app/controllers/abraham_histories_controller.rb]: Does this still work? Not ``@abraham_history.creator_id = current_user.id``?

File: app/helpers/abraham_helper.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    tours = Rails.configuration.abraham.tours["#{controller_name}.#{action_name}.#{I18n.locale}"]
+
+    unless tours
+      # How about the default locale?
+      tours = Rails.configuration.abraham.tours["#{controller_name}.#{action_name}.#{I18n.default_locale}"]
+    end
+
+    if tours
+      completed = AbrahamHistory.where(
+        creator_id: current_user, controller_name: controller_name,
```
Comment (seouri) [inline app/helpers/abraham_helper.rb]: Not ``creator_id: current_user.id``?

File: README.md (hunk 1 above)
Comment (jabbett) [inline README.md]: Me neither ;)

File: app/controllers/abraham_histories_controller.rb (hunk 2 above)
Comment (jabbett) [inline app/controllers/abraham_histories_controller.rb]: right, will push these commits shortly

File: app/helpers/abraham_helper.rb (hunk 3 above)
Comment (jabbett) [inline app/helpers/abraham_helper.rb]: done

**Pull request closed without merging**